.github/pull_request_template.md

```diff
@@ -5,10 +5,11 @@
 [List any issues this PR will resolve]
  
 ### Check List
+- [ ] All tests pass
```

Comment:
this should be spruced up but i moved this higher because not everything needed new tests but tests should pass